docs/source/using/index.adoc

```diff
@@ -6,6 +6,7 @@ This section describes how to manage your Minishift VM and interact with your
 OpenShift cluster. It also helps troubleshooting common issues.
 
 - link:../using/managing-minishift{outfilesuffix}[Managing Minishift]
+- link:../using/proxies{outfilesuffix}[HTTP/HTTPS Proxies]
```

Comment:
I had initially suggested proxy as a separate doc, but now looking at the changes, it  does not look good. I think we should move back the proxy piece back to the original doc.